smithy-model/src/main/java/software/amazon/smithy/model/node/NodeMapper.java

```diff
@@ -164,15 +164,17 @@ private static String createMessage(String property, String pointer, Class<?> in
          *
          * @param nodeType Node type being converted.
          * @param target The class to create from the Node.
+         * @param nodeMapper The NodeMapper being used to call the ObjectCreator.
```

Comment:
Could you move the changes to NodeMapper in smithy-model into a separate PR?

It'd be helpful to have those split out in the commit history?